### PR TITLE
Fix build process

### PR DESCRIPTION
**Pull request by MangelMaxime**
I updated the instructions from the readme and fixed the process to work under windows.

### CODE REVIEW

File: restore.cmd [hunk 1]
```diff
@@ -1,4 +1,3 @@
-yarn install
-dotnet restore Fable.proj
-dotnet restore ozmo
-dotnet restore react-todomvc
\ No newline at end of file
+call yarn install
+call dotnet restore Fable.proj
+call dotnet restore Fable.Samples.sln
```
Comment (alfonsogarciacaro) [inline restore.cmd:3]: Oh! 😮 So we have to use the solution name explicitly, that's why I couldn't restore all the projects of the solution at once (I was just typing `dotnet restore`). Thanks for the tip!

File: README.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ run `dotnet fable npm-run build`.
 ## Adding a new sample
 
 - Create a new folder with the sample (take current samples as reference)
-- Add the project to the `Fable.Samples.sln` solution (CLI: `dotnet sln add my-sample`)
+- Add the project to the `Fable.Samples.sln` solution (CLI: `dotnet sln Fable.Samples.sln add my-sample.fsproj`)
 - Add the sample to the restore scripts
```
Comment (alfonsogarciacaro) [inline README.md]: As you fixed my mistake and it's not necessary to restore each project explicitly, we can remove this line: "Add the sample to the restore scripts" :+1:

**Review by alfonsogarciacaro — COMMENTED** [review]
Awesome, thanks for the fix! Just one thing, we shouldn't need the paket.bootstrapper file as we are using [Paket magic mode](https://fsprojects.github.io/Paket/bootstrapper.html#Magic-mode).

File: README.md (hunk 2 above)
Comment (MangelMaxime) [reply README.md]: Sure will fix it

**Comment by MangelMaxime** [comment]
True about magic mode. I just didn't see paket.exe in the directory. Will fix it to use the magic mode so 😊

Thanks for the review

**Comment by alfonsogarciacaro** [comment]
Ups, true, I forgot to remove the `.paket/paket.exe` line from .gitignore 😅 Could you please do it for this PR, then add paket.bootsrapper.exe (make sure it's the one for Paket 5 beta) and rename it paket.exe? Thanks!

**Comment by MangelMaxime** [comment]
Sure no problem :)

**Comment by MangelMaxime** [comment]
@alfonsogarciacaro This PR should be ready to be merge.

**Pull request merged**